test/loopback-dl.test.js

```diff
@@ -260,6 +260,23 @@ describe('ModelBuilder', function() {
     follow.should.have.property('id');
     assert.deepEqual(follow.id, {followerId: 1, followeeId: 2});
   });
+
+  it('instantiates model from data with no constructor', function(done) {
```

Comment:
@thaerlabs Thank you for the test case. This test case only tests for the `happy case`. Could you add another case for the `unhappy case` (i.e when the data has a `constructor` property)?